_docs/changelog.md

```diff
@@ -8,7 +8,11 @@ permalink: /docs/changelog/
 
 Current version of the API is `1`
 
-Current version of the SDKs are `1.30.0`
+Current version of the SDKs are `1.30.1`
+
+1.30.1
```

Comment:
This will be `1.30.4` (the real latest at the moment is `1.30.3`, it's just not been updated here)

Reply:
This version should match the one in the specification.